src/client/components/SuggestedRoutes/SuggestedRoutesList.js

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from "react";
+import SuggestedRouteCard from "./SuggestedRouteCard";
+
+export default class SuggestedRoutesList extends Component {
+  state = {
+    selectedId: null
+  };
+
+  handleClick = id => {
+    this.setState({
+      ...this.state,
```

Comment:
why you are spreading the old state here before assigning a new one?

Reply:
I always recommend doing that. You have to pass a full copy of the state right? To preserve immutability?

I know in this case we only have `selectedId` so it won't make a difference, but in case more variables are added to the state later.

Reply:
Looks like I picked up this habit from using Redux. You should safely be able to remove line 10.

Reply:
but setState does this out of the box :D

Reply:
@zkwsk what is the final decision!

Reply:
It's line 11 you can remove.